### PR TITLE
Adding housing schema

### PR DESCRIPTION
**Pull request by JTA2020**
This is to supplement the core schema with the extended attributes to support the housing initiative

### CODE REVIEW

File: schemas/definitions/housing-permit-schema.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      "description": "The type of dwelling units being developed, e.g, Single-family, Multi-family, Secondary Unit, Other (optional).",
+      "enum": [
+        "Single-family",
+        "Multi-family", 
+        "Secondary Unit",
+        "Duplexes",
+        "Apartments",
+        "Condominiums",
+        "Other"
+      ],},
```
Comment (annikaLiving) [inline schemas/definitions/housing-permit-schema.json]:
```suggestion
      ]},
```

File: schemas/definitions/housing-permit-schema.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        "type": "string",
+        "description": "A value indicating whether the application's purpose pertains to Social Housing, i.e., Y, N, U.",
+        "enum": [
+          "Y",
+          "N", 
+          "U"
+        ]},
+    },
+  
+  },
```
Comment (annikaLiving) [inline schemas/definitions/housing-permit-schema.json]:
```suggestion
```

File: schemas/definitions/housing-permit-schema.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        "Other"
+      ],},
+      "housing_dwelling_type_code": {
+        "type": "string",
+        "description": "The code for the type of dwelling units being developed, e.g, SFD, MFD, SU, O.",
+        "enum": [
+          "SFD",
+          "MFD", 
+          "SU"
+        ]},
```
Comment (annikaLiving) [inline schemas/definitions/housing-permit-schema.json]:
```suggestion
          ]},
```

**Review by annikaLiving — CHANGES_REQUESTED** [review]
Made a few space and bracket updates.

**Review by annikaLiving — APPROVED** [review]
Looks good and aligned.

**Review by annikaLiving — APPROVED** [review]
Updates now align the items

**Review by mishraomp — APPROVED** [review]
LGTM

**Pull request merged**